Стилизуем письма

katanaevav opened the pull request:
---
:mortar_board: [Стилизуем письма](https://up.htmlacademy.ru/email/1/user/428115/tasks/4)

:boom: https://htmlacademy-email.github.io/428115-play-html-1/3/

katanaevav committed:
Во всех трех письмах логотип сделан ссылкой на htmlacademy.ru

katanaevav committed:
Исправлены адреса Полезных ссылок в подвале

katanaevav committed:
Исправлены адреса социальных сетей

katanaevav committed:
Исправлены ссылки в соответствии с ТЗ

katanaevav committed:
Обнуляем отступы у тега body для всех трех писем

katanaevav committed:
Приглашение на курс: установлен фон для общего контейнера

katanaevav committed:
Приглашение на курс: создана сетка с отступами и фонами для блоков

katanaevav committed:
Приглашение на курс: испарвление фонового цвета

katanaevav committed:
Приглашение на курс: добавлена ширина таблиц

katanaevav committed:
Приветственное письмо: создана сетка с отступами и фонами для блоков

katanaevav committed:
Приветственное письмо: исправлены отступы в блоке приветствия

katanaevav committed:
Приветственное письмо: подправлена центровка меню

katanaevav committed:
Акция: создана сетка до раздела курсов, учавствующих в акции

katanaevav committed:
Акция: создана сетка раздела Отзыв о кусре

katanaevav committed:
Акция: создана общая сетка разделов популярных курсов и приемуществ

katanaevav committed:
Акция: создана общая сетка разделов дополнена

katanaevav committed:
Акция: создана сетка разделов популярные курсы

katanaevav committed:
Акция: завершена общая сетка письма

katanaevav committed:
Акция: иправления для Outlook

katanaevav committed:
Акция: исправлены ошибки валидатора

katanaevav committed:
Акция: исправлена ошибка в теге

katanaevav committed:
Исправлены ошибки валидатора

gkarev commented:
Необходимо немного доработать сейв зону, и скорректировать ширину шаблона
вот так выглядит в макет
![image](https://user-images.githubusercontent.com/13588373/99879268-d342aa00-2c1c-11eb-9cf5-20d339cd3ef6.png)

видно что с боков есть по 15px отступы, и задан другой фон. сама ширина письма = 600px, а вот контентной части, меньше https://prnt.sc/vndxb2 

просьба, также сверить с другими макетами.

katanaevav committed:
Исправлены замечания

В 1 и 3 писмах исправлены ширины контентных областей.
Во 2 письме - ширина контентной области = ширине письма. Исправлен отступ сверху у меню. Добавлена горизонтальная линия между подвалом с текстом и контактами

katanaevav committed:
Акция: исправлен отступ перед блоком с контактами

katanaevav committed:
Приветственное письмо: исправлены ошибки валидатора

katanaevav committed:
Приветственное письмо: исправлен общий фон
